src/sgame/sg_bot_util.cpp

```diff
@@ -1860,7 +1860,7 @@ void BotFireWeaponAI( gentity_t *self )
 			}
 			break;
 		case WP_ALEVEL3_UPG:
-			if ( self->client->ps.ammo > 0 && distance > LEVEL3_CLAW_UPG_RANGE )
+			if ( self->client->ps.ammo > 0 && distance > LEVEL3_CLAW_UPG_RANGE && distance > (4.0f/3.0f * BG_Missile(MIS_BOUNCEBALL)->splashRadius) )
```

Comment:
What's the 4.0/3.0 for? Let's also add comments about this

Reply:
commented